.github/workflows/build-dockerfile.yml

```diff
@@ -0,0 +1,23 @@
+name: Control Dockerfiles
+on:
+  push:
+    paths:
+      - '**'
+      - '!README.md'
+      - '!LICENSE'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["armv7hf", "aarch64"]
+    env:
+      EXREPO: build-test
```

Comment:
Looks slightly inconsistent that these are capitalized, but maybe there's a good reason for it.

Reply:
Copied from example repos and there was an idea which doesn't make sense here now